code/modules/projectiles/guns/launcher/syringe_gun.dm

```diff
@@ -63,7 +63,7 @@
 
 /obj/item/weapon/gun/launcher/syringe
 	name = "syringe gun"
-	desc = "A spring loaded rifle designed to fit syringes, designed to incapacitate unruly patients from a distance."
+	desc = "A spring-loaded rifle designed to fit syringes designed to incapacitate unruly patients from a distance."
```

Comment:
this whole line is bad, having "designed" twice in succession like that sounds wrong, try rewriting it completely